### PR TITLE
Easy mode for hiding all the params except primaryInput

### PR DESCRIPTION
**Pull request by gokaykucuk**
We still have to carry it to state but for now we can just hide the other parameters.

### CODE REVIEW

File: app/src/components/Form.js [hunk 1]
```diff
@@ -58,9 +59,16 @@ export default React.memo(FormView)
 function getFormInputs(ipfs, metadata) {
     if ((metadata === undefined) || (metadata === null)) return;
     ipfs = ipfs || {};
-
-    return Object.fromEntries(Object.entries(metadata.form.properties).map(
-            ([formKey, prop]) => [formKey, formKey in ipfs ? { ...prop, "default": ipfs[formKey] } : prop]))
+    const allParams = Object.fromEntries(Object.entries(metadata.form.properties).map(
```
Comment (gokaykucuk) [inline app/src/components/Form.js:62]: @voodoohop real change is here, the rest got auto-fixed by my ide.

**Comment by voodoohop** [comment]
I see 4 options to enable this:
1) a query parameter that enables "easy" mode or the opposite to enable "expert" mode
2) another prefix e.g. /e/[hash] is the easy mode, /p/[hash] is the (current) expert mode
3) localStorage
4) we don't code an easy mode into the pollinations code but have expert and easy notebooks using different notebook.ipynb's so one would have only one parameter and the expert one would make more available using the built-in colab parameters

**Comment by voodoohop** [comment]
> I see 4 options to enable this:
> 
> 1. a query parameter that enables "easy" mode or the opposite to enable "expert" mode
> 2. another prefix e.g. /e/[hash] is the easy mode, /p/[hash] is the (current) expert mode
> 3. localStorage
> 4. we don't code an easy mode into the pollinations code but have expert and easy notebooks using different notebook.ipynb's so one would have only one parameter and the expert one would make more available using the built-in colab parameters

@gokaykucuk which one do you think makes most sense?

**Comment by gokaykucuk** [comment]
> I see 4 options to enable this:
> 
> 1. a query parameter that enables "easy" mode or the opposite to enable "expert" mode
> 2. another prefix e.g. /e/[hash] is the easy mode, /p/[hash] is the (current) expert mode
> 3. localStorage
> 4. we don't code an easy mode into the pollinations code but have expert and easy notebooks using different notebook.ipynb's so one would have only one parameter and the expert one would make more available using the built-in colab parameters

@voodoohop I think a combination of localStorage and query parameter makes sense. We can bind the query parameter to a value in localStorage and use it as a way of switching modes.

ex:

user1 navigates to pollinations first time => redirect to easy mode with ?mode=easy and store mode=easy in localStorage
user1 navigates to pollinations with mode=expert in query => store mode=expert in localStorage
user1 navigates to pollinations with mode=expert in localStorage => update url to ?mode=expert.

**Comment by voodoohop** [comment]
Another option would be simple collapsible advanced options. I kind of like it because it adds no state complexity

advanced-options-collapsed
advanced-options-expanded

**Comment by voodoohop** [comment]
@gokaykucuk I really like just having the expandable "> advanced option". if you think it's good enough for now I'd suggest we go ahead with it. I can implement it and merge or if you have time.

**Pull request closed without merging**